build_extopus_deps.sh

```diff
@@ -2,11 +2,9 @@
 
 . `dirname $0`/sdbs.inc
 
-if python -V 2>&1 | egrep -q '2.[5-9]'; then
-:
-else
-   simplebuild http://www.python.org/ftp/python/2.7.1 Python-2.7.1.tgz
-fi
+# qooxdoo needs python 2.5 or newer
+${SDBS_SCRIPTS_DIR}/check_or_build_python2_5.sh
```

Comment:
Because qooxdoo and glib check if python is 2.5 or higher.
On Jan 15, 2015 8:10 AM, "Tobias Oetiker" <notifications@github.com> wrote:

> In build_extopus_deps.sh
> <https://github.com/oetiker/sdbs/pull/21#discussion-diff-22993703>:
>
> > @@ -2,11 +2,9 @@
> >
> >  . `dirname $0`/sdbs.inc
> >
> > -if python -V 2>&1 | egrep -q '2.[5-9]'; then
> > -:
> > -else
> > -   simplebuild http://www.python.org/ftp/python/2.7.1 Python-2.7.1.tgz
> > -fi
> > +# qooxdoo needs python 2.5 or newer
> > +${SDBS_SCRIPTS_DIR}/check_or_build_python2_5.sh
>
> any particular reason for calling the script 2_5 ? why not just
> check_or_build_python2.sh
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oetiker/sdbs/pull/21/files#r22993703>.
>